test/test_transformers.py

```diff
@@ -3670,7 +3589,7 @@ def setUpClass(cls):
         cls.module = torch.utils.cpp_extension.load(
             name="custom_device_extension",
             sources=[
-                "cpp_extensions/open_registration_extension.cpp",
+                f"{'test/' if not os.getcwd().endswith('test') else ''}cpp_extensions/open_registration_extension.cpp",
```

Comment:
What's going on here?

Reply:
I think this is because these tests work in CI but dont work locally